src/main/webapp/app/exam/exam-scores/exam-scores.component.html

```diff
@@ -90,17 +94,11 @@ <h4>
             </tbody>
         </table>
 
-        <!-- Histogram Table -->
-        <table class="table table-bordered table-sm table-clear-width mb-3">
-            <thead class="thead-dark">
-                <th *ngFor="let _ of histogramData; let i = index; last as isLast">[{{ i * binWidth }},{{ (i + 1) * binWidth }}{{ isLast ? ']' : ')' }}</th>
-            </thead>
-            <tbody>
-                <td *ngFor="let noOfStudents of histogramData">
-                    {{ noOfStudents }} ({{ noOfExamsFiltered ? roundAndPerformLocalConversion((noOfStudents * 100) / noOfExamsFiltered, 2, 2) : '-' }}%)
-                </td>
-            </tbody>
-        </table>
+        <div style="display: block;">
+            <!-- Histogram -->
+            <canvas baseChart height="400" [datasets]="barChartData" [labels]="barChartLabels" [options]="barChartOptions" [legend]="barChartLegend" [chartType]="barChartType">
+            </canvas>
+        </div>
 
         <!-- Exercise Statistics Table -->
         <table class="table table-bordered table-sm body-striped">
```

Comment:
This table has a styling issue on mobile: 
<img width="387" alt="Bildschirmfoto 2020-08-19 um 13 13 40" src="https://user-images.githubusercontent.com/13920539/90628470-f6288100-e21d-11ea-8cee-56d52856807c.png">
Wrapping the `<table>` element inside a `<div class="table-responsive">` like the table below should fix this.

Reply:
Fixed it in 2f2f10706a14b874cd33dc46636fb05827b7411f